GuruLoader/GuruLoader.cs

```diff
@@ -80,6 +80,25 @@ public class DisplayPortfolio {
     public IEnumerable<DisplayPosition> Positions { get; set; }
 }
 
+public class HyperPosition {
+    public string Name { get; set; }
+    public string ClassTitle { get; set; }
+    public string Cusip { get; set; }
+    public string PutCall { get; set; }
+    public double PercOfPortfolio { get; set; }
+    public int NumberGurusOwning { get; set; }
+    public int NumberGurusBuying { get; set; }
+    public int NumberGurusSelling { get; set; }
+    public IList<string> Gurus { get; set; }
+}
+
+public class HyperPortfolio {
+    public DateTime EndQuarterDate { get; set; }
+    public int NumberOfGurus { get; set; }
```

Comment:
Is there value in knowing which Gurus a HyperPortfolio covers? Or is it good enough to know the count and get the specific gurus from the HyperPositions?